src/main/java/com/crud/tasks/controller/TaskController.java

```diff
@@ -0,0 +1,48 @@
+package com.crud.tasks.controller;
+
+import com.crud.tasks.domain.TaskDto;
+import com.crud.tasks.mapper.TaskMapper;
+import com.crud.tasks.service.DbService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@RestController
+@RequestMapping("/v1/task")
+public class TaskController {
+    @Autowired
+    private DbService service;
+    @Autowired
+    private TaskMapper taskMapper;
+
+    @RequestMapping(method = RequestMethod.GET, value = "")
+    public List<TaskDto> getTasks() {
+        return taskMapper.mapToTaskDtoList(service.getAllTasks());
+    }
+
+    @RequestMapping(method = RequestMethod.GET, value = "/{taskId}")
+    public TaskDto getTask(@PathVariable("taskId") String taskId) {
+        return new TaskDto((long) 1, "test title", "test_content");
+    }
+
+    @RequestMapping(method = RequestMethod.DELETE, value = "/{taskId}")
+    public HttpStatus deleteTask(@PathVariable("taskId") String taskId) {
+        return HttpStatus.OK;
```

Comment:
in case of delete, http 204 is rather returned - because we don't return any additional body.